core/templates/dev/head/domain/collection/CollectionUpdateService.js

```diff
@@ -38,11 +38,6 @@ oppia.constant(
   'COLLECTION_NODE_PROPERTY_PREREQUISITE_SKILL_IDS', 'prerequisite_skill_ids');
 oppia.constant(
   'COLLECTION_NODE_PROPERTY_ACQUIRED_SKILL_IDS', 'acquired_skill_ids');
-// Note: these properties are deprecated.
```

Comment:
Ah -- I'll add a comment to say that, but not actually declare the constants so that there's no possibility of them being imported by accident. Is that OK?